test/api_integration/apis/data_views/data_views_crud/delete_data_view/main.ts

```diff
@@ -41,21 +41,6 @@ export default function ({ getService }: FtrProviderContext) {
 
           expect(response4.status).to.be(404);
         });
```

Comment:
I removed this test because there was a weird relationship between it at and the versioned router. The versioned router would provide a weird message even though the task completed successfully and this is the circumstance under which this test would pass. Now the api returns no body and I'm no longer testing for its emptiness - which wasn't registering as emptiness in this test for some reason.

Reply:
What was it registering as instead?

Reply:
`res.ok()` on the sender side results in an empty object for the body